client/assets/js/meter.js

```diff
@@ -32,10 +32,14 @@ var target = {
 //     }
 // });
 // sound.audible = false;
-function handleClickHandlers(){
-  $('.range-indicator').on('click', function(){
+
+
+function handleEventHandlers(){
+  $('.range-indicator').on('click touch', function(){ //knob switch
```

Comment:
I still think this should be doable as a callback, rather than an anonymous function. Your call though.